conda/shell/Scripts/activate.bat

```diff
@@ -1 +1,25 @@
+@REM Test first character and last character of %1 to see if first character is a "
+@REM   but the last character isn't.
+@REM This was a bug as described in https://github.com/ContinuumIO/menuinst/issues/60
+@REM When Anaconda Prompt has the form
+@REM   %windir%\system32\cmd.exe "/K" "C:\Users\builder\Miniconda3\Scripts\activate.bat" "C:\Users\builder\Miniconda3"
+@REM Rather than the correct
+@REM    %windir%\system32\cmd.exe /K ""C:\Users\builder\Miniconda3\Scripts\activate.bat" "C:\Users\builder\Miniconda3""
+@REM this solution taken from https://stackoverflow.com/a/31359867
+@set "_c1=%1"
+@set _c1f=%_c1:~0,1%
+@set _c1l=%_c1:~-1%
+@set _c1f=%_c1f:"=+%
+@set _c1l=%_c1l:"=+%
+@set _c1=
+
+@if "%_c1f%"=="+" if NOT "%_c1l%"=="+" (
+    @CALL "%~dp0..\Library\bin\conda.bat" activate
```

Comment:
This will activate the root/base environment correct?  Is that the intended behavior?  Do we want to echo any type of warning message about dropping the command line parameters?

Reply:
The point of this is to give conda 4.4 the behavior conda 4.3 and earlier had.  If you had a directory `C:\Users\build er\Miniconda3`, and were to do something like

    %windir%\system32\cmd.exe "/K" "C:\Users\build er\Miniconda3\Scripts\activate.bat" "C:\Users\build er\Miniconda3"

and also put `echo %*` at the top of `activate.bat`, you get 

    "C:\Users\build

as output.  Which is total crap.  But conda 4.3 and earlier just swallowed that, and activated the root environment.  That's all we're trying to accomplish here.